### ISSUE TITLE
Ask to modify input question

### ISSUE THREAD

**Issue by 1109726679** [sjtuplayer/anomalydiffusion#106]
Hello author, I am currently reproducing your paper and want to change the input data from the test set to the training set. I have tried many methods but have not been successful, so I have to bother you and ask for your advice. Please feel free to give me some advice.
